packages/xcoobee-cookie-kit-react/src/CookieKitContainer.js

```diff
@@ -103,7 +103,7 @@ export default class CookieKitContainer extends React.PureComponent {
     expirationTime: PropTypes.number,
     hideBrandTag: PropTypes.bool,
     hideOnComplete: PropTypes.bool,
-    position: PropTypes.oneOf(positions),
+    position: PropTypes.string,

```

Comment:
I don't understand why this was changed. Would you please explain?

Reply:
@maklyakov 
did you resolve this with Dan ?